src/lib/run_environment.ml

```diff
@@ -28,10 +28,11 @@ module Tool = struct
       [@@@ocaml.warning "-11"]
       type t = [
         | `Bwa of [ `V_0_7_10 ]
-        | `Custom of string * string
+        (* A tool that is manually installed by retrieving source or binary.*)
```

Comment:
So it is for the convenience of `Definition.t`. Are there other parameters (fields) that a tool needs? Perhaps changing it to a simpler record